packages/react-dom/src/__tests__/ReactDOMSuspensePlaceholder-test.js

```diff
@@ -83,25 +83,25 @@ describe('ReactDOMSuspensePlaceholder', () => {
           <div ref={divs[1]}>
             <AsyncText ms={500} text="B" />
           </div>
-          <div style={{display: 'block'}} ref={divs[2]}>
+          <div style={{display: 'span'}} ref={divs[2]}>
```

Comment:
?

Reply:
I changed it to `span` since `block` is the default display for divs.

Reply:
Ahaha I meant `inline`.

Reply:
Thanks, brain!

Reply:
I was talking to my brain, not you.

Reply:
But `!important` only accept from `style.setProperty` is really first time killer except holder of CSS degree😅
Thanks find out this @acdlite 😀

Reply:
Following up Andrew to learn CSS. :smile：